web/lib/queries/common/pageContentFields.ts

```diff
@@ -578,6 +578,16 @@ _type == "keyNumbers" =>{
       }
     },
   },
+  _type == "imageForText" => {
+    "type": _type,
+    "id": _key,
+    "content": content[]{..., ${markDefs}},
+    "aspectRatio": coalesce(aspectRatio, '10:3'),
```

Comment:
aspect ratio 10:3 is not in allowed list configured ( 16:9 or full width )

Reply:
Nice catch